### ISSUE TITLE
Cannot focus closed markdown cell when focus lost

### ISSUE THREAD

**Issue by webczat** [microsoft/vscode#107581]
Version: 1.50.0-insider
Commit: 403a6895f401c397933ec5c4f7d89432a33509b6
Date: 2020-09-18T07:13:12.194Z
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 5.8.10-arch1-1

It seems to me that I cannot focus closed markdown cells, or it is not accessible with screenreader.
I created both markdown and c# cells, closed both of them and lost focus by tabbing around. I cannot return to the markdown editor or go directly to the rendered markdown, at least screenreader in browse mode does not see it is there. if I'd switch back to focus/forms mode, I cannot also press arrows and move between cells. However if I press tab long enough I seem to be able to land on the c# cell. If I remove this c# cell I cannot focus anything like cell list or rendered output of markdown cell, only browse mode seems to help at least when wanting to see the output.
CC: @isidorn

**Comment by isidorn** [comment]
Related discussion https://github.com/microsoft/vscode/issues/93896

Forwarding to @rebornix and @roblourens for Notebook

**Comment by roblourens** [comment]
Moving focus around the cell list should work basically the same as other lists in vscode, and the logic for editing cells is the same as in the new settings editor: enter should put the markdown cell back in edit mode and put focus in the editor. And escape moves focus back to the list and gets rid of the editor in a markdown cell.

When you say you "closed" the cells, what do you mean exactly?

**Comment by webczat** [comment]
the thing was I cannot refocus the list when I leave it, for example if I'd close all cells (means, press esc on all of them) then go to explorer, how would I go back... :) In markdown case tab does not reach markdown cell if it is not in edit mode

**Comment by roblourens** [comment]
I can tab back from the explorer to the list. The list gets focus and then I can continue navigating with arrow keys. It should be a tab target like other lists.

**Comment by webczat** [comment]
how is that announced with a screenreader? I will likely download newest insiders and retry.

**Comment by roblourens** [comment]
I'm not sure, is something announced for this when focusing other lists?

**Comment by webczat** [comment]
yes. but that is not exactly the question as I cannot say whether I focus the list but can't figure that out because accessibility is broken, or I don't focus the list because I have too old insiders or I do something wrong. so trying to actually figure that out

**Comment by roblourens** [comment]
When I focus the explorer, it just reads the focused row. When I focus a notebook row, it only reads "and one more item and 2 more items, group" which is definitely not useful.

**Comment by amunger** [comment]
This seems to work as expected now. Getting focus back to the cell list will announce something like "cell 1 of 2"
